pom.xml

```diff
@@ -1243,6 +1243,24 @@
                 <artifactId>simpleclient_servlet</artifactId>
                 <version>${prometheus.version}</version>
             </dependency>
+            <!-- https://mvnrepository.com/artifact/org.osgi/org.osgi.service.component -->
+            <dependency>
+                <groupId>org.osgi</groupId>
+                <artifactId>org.osgi.service.component</artifactId>
+                <version>${org.osgi.version}</version>
+            </dependency>
+            <!-- https://mvnrepository.com/artifact/org.osgi/org.osgi.service.component.annotations -->
+            <dependency>
+                <groupId>org.osgi</groupId>
+                <artifactId>org.osgi.service.component.annotations</artifactId>
```

Comment:
shall we move this to migration client , not to outer files and make it self contained.

Reply:
done